dask_sql/physical/rel/base.py

```diff
@@ -84,7 +86,9 @@ def assert_inputs(
         return [RelConverter.convert(input_rel, context) for input_rel in input_rels]
 
     @staticmethod
-    def fix_dtype_to_row_type(dc: DataContainer, row_type: "RelDataType"):
+    def fix_dtype_to_row_type(
+        dc: DataContainer, row_type: "RelDataType", join_type: str = None
```

Comment:
```suggestion
        dc: DataContainer, row_type: "RelDataType", join_type: str | None = None
```

Reply:
Based on your comment above think this should work

```suggestion
        dc: DataContainer, row_type: "RelDataType", join_type: Optional[str] = None
```